### PR TITLE
Fetch suitable chromedriver

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Avoids selenium-standalone diverging from the latest version of Chrome.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -128,6 +128,16 @@ chrome:
 		sudo ln -s "$$(which chromium)" $(CHROME_PATH); \
 	fi
 
+# https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+chromedriver: curl grep unzip chrome
+	if [[ -z "$$(which chromedriver)" ]]; then \
+		CHROME_VERSION=$$(google-chrome --version | grep -ioE "[0-9]+\.[0-9]+\.[0-9]+"); \
+		CHROMEDRIVER_VERSION=$$(curl https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$${CHROME_VERSION}); \
+		sudo curl https://chromedriver.storage.googleapis.com/$${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip -o $(CHROMEDRIVER_PATH).zip; \
```
Comment (Hexcles) [inline Makefile]:
```suggestion
		wget https://chromedriver.storage.googleapis.com/$${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip; \
```

File: Makefile [hunk 2]
```diff
@@ -128,6 +128,16 @@ chrome:
...
+		sudo unzip $(CHROMEDRIVER_PATH).zip -d $$(dirname $(CHROMEDRIVER_PATH)); \
```
Comment (Hexcles) [inline Makefile]:
```suggestion
		sudo unzip chromedriver_linux64.zip -d $$(dirname $(CHROMEDRIVER_PATH)); \
```

File: Makefile [hunk 3]
```diff
@@ -128,6 +128,16 @@ chrome:
...
+		sudo chmod +x $(CHROMEDRIVER_PATH); \
```
Comment (Hexcles) [inline Makefile]:
```suggestion
```

File: Makefile [hunk 4]
```diff
@@ -128,6 +128,16 @@ chrome:
 		sudo ln -s "$$(which chromium)" $(CHROME_PATH); \
 	fi
 
+# https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+chromedriver: curl grep unzip chrome
```
Comment (Hexcles) [inline Makefile]:
```suggestion
chromedriver: wget unzip chrome
```

File: Makefile [hunk 5]
```diff
@@ -164,11 +174,13 @@ sys_deps: curl gpg node gcloud git
 
 curl: apt-get-curl
 git: apt-get-git
+grep: apt-get-grep
```
Comment (Hexcles) [inline Makefile]:
```suggestion
```

`grep` is required by POSIX so should be available everywhere.

**Comment by Hexcles** [comment]
The driver & browser versions now match, but we're still seeing the same error: https://travis-ci.com/web-platform-tests/wpt.fyi/jobs/179335988#L2690

**Comment by Hexcles** [comment]
I'm fairly certain this is part of the problem, because `go_chrome_test` went from passing to failing on master between these two runs:
https://travis-ci.com/web-platform-tests/wpt.fyi/jobs/178336711
https://travis-ci.com/web-platform-tests/wpt.fyi/jobs/179172652
And the Chrome version did go from 71 to 72.

However, there might be something else missing. Does it work locally inside Docker?

File: Makefile (hunk 3 above)
Comment (lukebjerring) [reply Makefile]: I think we still need this; `which chromedriver` fails without it.

File: Makefile [hunk 6]
```diff
@@ -128,6 +128,16 @@ chrome:
 		sudo ln -s "$$(which chromium)" $(CHROME_PATH); \
 	fi
 
+# https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+chromedriver: curl grep unzip chrome
+	if [[ -z "$$(which chromedriver)" ]]; then \
```
Comment (lukebjerring) [inline Makefile]:
```suggestion
	if [[ ! -f "$(CHROMEDRIVER_PATH)" ]]; then \
```

**Comment by lukebjerring** [comment]
Yes, it does run locally fine (inside docker).

**Comment by Hexcles** [comment]
Oh no you totally butchered the indentation... Let me try to fix it...

**Pull request merged**